scripts/better-tables.js

```diff
@@ -34,6 +34,14 @@ export class BetterTables {
         await lootChatCard.createChatCard(tableEntity);
     }
 
+    async _getStoryResults(tableEntity){
```

Comment:
hi, can you change the signature without the starting _
this class is providing api for using the module in macro / other modules so will be in line with the rest.
I m fine with the rest 👍🏻

Reply:
awesome, will do!